tests/api/test_guidelines.py

```diff
@@ -24,8 +24,7 @@
 from parlant.core.guidelines import Guideline, GuidelineContent, GuidelineStore
 from parlant.core.services.tools.service_registry import ServiceRegistry
 from parlant.core.tools import LocalToolService, ToolId
-from tests.core.services.tools.test_openapi import OPENAPI_SERVER_URL, rng_app, run_openapi_server
-from tests.core.services.tools.test_plugin_client import run_service_server
+from tests.test_utilities import OPENAPI_SERVER_URL, rng_app, run_openapi_server, run_service_server
```

Comment:
I see you've probably deliberately removed the blank lines and not accidentally, so maybe there's something to discuss here. What's your take on it?